Quadriga/src/main/java/edu/asu/spring/quadriga/domain/factory/workbench/ITextFileFactory.java

```diff
@@ -0,0 +1,22 @@
+package edu.asu.spring.quadriga.domain.factory.workbench;
+
+import org.springframework.stereotype.Service;
+
+import edu.asu.spring.quadriga.domain.workspace.ITextFile;
+
+
+/**
+ * @author Nischal Samji
+ * Generates Text File Objects
+ *
+ */
+@Service
+public interface ITextFileFactory {
+
+    /**
+     * @return
+     * 		returns a TextFile Object
+     */
+    public abstract ITextFile createTextFileObject();
```

Comment:
description of what the method does. Does it fill any fields? Or just constructs an empty object?